src/code/z_kankyo.c

```diff
@@ -1538,27 +1648,27 @@ void Environment_DrawLensFlare(GlobalContext* globalCtx, EnvironmentContext* env
                 gDPSetColorDither(POLY_XLU_DISP++, G_CD_DISABLE);
 
                 if (!(isOffScreen ^ 0)) {
-                    Math_SmoothStepToF(&envCtx->unk_84, alpha * alphaScale, 0.5f, 50.0f, 0.1f);
+                    Math_SmoothStepToF(&envCtx->lensFlareFillAlpha, alpha * alphaScale, 0.5f, 50.0f, 0.1f);
                 } else {
-                    Math_SmoothStepToF(&envCtx->unk_84, 0.0f, 0.5f, 50.0f, 0.1f);
+                    Math_SmoothStepToF(&envCtx->lensFlareFillAlpha, 0.0f, 0.5f, 50.0f, 0.1f);
                 }
 
                 temp = colorIntensity / 120.0f;
                 temp = CLAMP_MIN(temp, 0.0f);
 
                 gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, 255, (u8)(temp * 75.0f) + 180, (u8)(temp * 155.0f) + 100,
-                                (u8)envCtx->unk_84);
+                                (u8)envCtx->lensFlareFillAlpha);
                 gDPFillRectangle(POLY_XLU_DISP++, 0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1);
```

Comment:
Pretty sure this is glare implementation
(in my understanding "lens flare" is the localized spots of lights when looking at the sun,
"glare" is the overall blindness induced)

Would suggest
`lensFlareFillAlpha` -> `glareAlpha`
`screenFillAlpha` -> `glareStrength`
`alphaScale` -> `glareAlphaScale`
also `alphaScaleTarget` -> `lensFlareAlphaScaleTarget`